babel.config.js

```diff
@@ -1,83 +1,20 @@
-const TARGETS = {
-  chrome: '60',
-  edge: '15',
-  firefox: '53',
-  ios: '10.3',
-  safari: '10.1',
-  node: '8'
-};
+const getBabelConfig = require("ocular-dev-tools/config/babel.config");
```

Comment:
Helpful to see it like this, I didn't realize that we do get to import the base config from ocular, that's great!

The override process is still managed by ocular, I guess that has both advantages and disadvantages.

I guess it does provides the best of both worlds, the app can still manually override before exporting.